app/views/application/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1><%= @application.name %></h1>
+
+<h3> Street Address: <%= @application.street_address %></h3>
+<h3> City: <%= @application.city %></h3>
+<h3> State: <%= @application.state %></h3>
+<h3> Zip Code: <%= @application.zip_code %></h3>
+<h3> Description why you should have a dog: <%= @application.description %></h3>
+<h3> Add a Pet to this Application </h3>
+<%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |f| %>
+  <%= f.label :search, "Search for pets by name" %>
+  <%= f.text_field :search %>
+  <%= f.submit "submit" %>
+<% end %>
+<% if @search_results.any? %>
+  <h3> Search Results: </h3>
+  <ul>
+    <% @search_results.each do |pet| %>
+      <li><%= "#{pet.name}" %></li>
+      <li><%= button_to 'Adopt this Pet', "/pets/#{pet.id}", method: :get %></li>
```

Comment:
As I mentioned in the comment in the PetApplicationsController, the most RESTful design for this functionality would be to route this button to that joins controller, and specifically to a create action for that joins controller (so the button method could be POST instead). A more RESTful path for that would also include both pets and applications, since you're creating a relationship between these 2 existing entities, i.e. creating a joins record. Consider either of these paths, both of which are RESTful:
`post "/applications/:application_id/pets/:pet_id", to "pet_applications#create"`
`post "/pet_applications", to "pet_applications#create"`

The benefit of the first option is that you can pass IDs in the path